### PR TITLE
License key

### PR DESCRIPTION
**Pull request by nerpaula**
- added info about license key when upgrading to Enterprise edition
- added info about license key for Kubernetes deployments

### CODE REVIEW

**Review by Simran-B — CHANGES_REQUESTED** [review]

File: 3.7/deployment-kubernetes-usage.md [hunk 1]
```diff
@@ -68,15 +68,30 @@ You can find the latest release of the ArangoDB Kubernetes Operator
...
+kubectl create secret generic arango-license-key --from-literal=token-v2="<license-string>"
+```
+
 Once the operator is running, you can create your ArangoDB database deployment
 by creating a `ArangoDeployment` custom resource and deploying it into your
-Kubernetes cluster.
-
+Kubernetes cluster. 
+   
 For example (all examples can be found [in the kube-arangodb repository](https://github.com/arangodb/kube-arangodb/tree/master/examples){:target="_blank"}):
```
[the remark this answers is not in the document]
Comment (nerpaula) [reply 3.7/deployment-kubernetes-usage.md]: For some reason, the checks have failed after applying this suggestion.

[the remark this answers is not in the document]
Comment (Simran-B) [reply 3.7/deployment-kubernetes-usage.md]:
```
/opt/build/repo/_plugins/versions/version.rb:45:in `<=>': undefined method `version' for nil:NilClass (NoMethodError)
```
This error occurs every so often, not sure why, but it has nothing to do with the content changes. You can push an empty commit (`git commit --allow-empty -m "Re-trigger Netlify"`) which should fix it.

**Review by Simran-B — APPROVED** [review]

File: 3.7/deployment-kubernetes-usage.md [hunk 2]
```diff
@@ -64,19 +64,34 @@ kubectl apply -f $URLPREFIX/arango-deployment-replication.yaml
 ```
 
 You can find the latest release of the ArangoDB Kubernetes Operator
-[in the kube-arangodb repository](https://github.com/arangodb/kube-arangodb/releases/latest){:target="_blank"}.
+in the [kube-arangodb repository](https://github.com/arangodb/kube-arangodb/releases/latest){:target="_blank"}.
 
 ## ArangoDB deployment creation
 
+After deploying the latest ArangoDB Kubernetes operator, use the command below to deploy your license key as a secret:
```
Comment (joerg84) [inline 3.7/deployment-kubernetes-usage.md:71]: Should mention it is EE only, not community

Comment (joerg84) [reply 3.7/deployment-kubernetes-usage.md:71]: And potentially a link to general LK documentation

File: 3.7/deployment-kubernetes-usage.md [hunk 3]
```diff
@@ -64,19 +64,34 @@ kubectl apply -f $URLPREFIX/arango-deployment-replication.yaml
...
 by creating a `ArangoDeployment` custom resource and deploying it into your
 Kubernetes cluster.
 
-For example (all examples can be found [in the kube-arangodb repository](https://github.com/arangodb/kube-arangodb/tree/master/examples){:target="_blank"}):
+For example (all examples can be found in the [kube-arangodb repository](https://github.com/arangodb/kube-arangodb/tree/master/examples){:target="_blank"}):
 
 ```bash
 kubectl apply -f examples/simple-cluster.yaml
 ```
+Additionally, you can specify:
```
Comment (joerg84) [inline 3.7/deployment-kubernetes-usage.md:86]: Again just a quick note on EE only

File: 3.9/upgrading-community-to-enterprise.md [hunk 4]
```diff
@@ -28,6 +28,9 @@ and installation of the Enterprise package. The upgrade can be done in a
 description of the two methods. Refer to the sections below for a detailed
 procedure.
 
+The Enterprise Edition of ArangoDB requires a license to activate the Enterprise Edition features.
```
Comment (joerg84) [inline 3.9/upgrading-community-to-enterprise.md:31]: Technically it is not just about activating the features, but running EE (i.e., an EE without a license key does not just become a community version, but does not run at all)

**Review by joerg84 — APPROVED** [review]
Thanks! Overall great, just left minor comments.

**Pull request merged**